android-ndk-r16b-api-19-armeabi-v7a-neon-hid-sections-lto.cmake

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2015-2018, Ruslan Baratov
+# Copyright (c) 2015-2018, David Hirvonen
+# Copyright (c) 2015-2018, Alexandre Pretyman
+# All rights reserved.
+
+if(DEFINED POLLY_ANDROID_NDK_R16B_API_19_ARMEABI_V7A_NEON_HID_SECTIONS_LTO_CMAKE_)
+  return()
+else()
+  set(POLLY_ANDROID_NDK_R16B_API_19_ARMEABI_V7A_NEON_HID_SECTIONS_LTO_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(ANDROID_NDK_VERSION "r16b")
+set(CMAKE_SYSTEM_VERSION "19")
+set(CMAKE_ANDROID_ARCH_ABI "armeabi-v7a")
+set(CMAKE_ANDROID_ARM_NEON TRUE)
+set(CMAKE_ANDROID_ARM_MODE TRUE) # 32-bit ARM
+set(CMAKE_ANDROID_NDK_TOOLCHAIN_VERSION "4.9") # <major>.<minor>: GCC of specified version
+set(CMAKE_ANDROID_STL_TYPE "c++_static") # LLVM libc++ static
```

Comment:
Same here, `libcxx` needed in name. Actually are they compatible?

Reply:
It seems so.  I wasn't sure about this until testing it actually.